crates/gpui/src/window.rs

```diff
@@ -3527,6 +3529,14 @@ impl WindowContext<'_> {
     }
 }
 
+#[cfg(target_os = "linux")]
+impl WindowContext<'_> {
```

Comment:
I actually wonder if we should implement this for all platforms. macOS has a similar API we could use:

https://developer.apple.com/documentation/appkit/nstextinputcontext/1535165-invalidatecharactercoordinates?language=objc

Reply:
Not sure about macOS but I don't think this is needed for Windows, because the enumerated Input Methods appear in a fixed location in the taskbar.

Reply:
Alright, I think it would be better to move this into the overall platform trait. Perhaps with a default, noop, implementation. I can come through and implement the macOS version later :)